### PR TITLE
[test-only][ful-ci] enable pagination test

### PR DESCRIPTION
**Pull request by ScharfViktor**
related https://github.com/owncloud/ocis/issues/9648#issuecomment-2239407618

### CODE REVIEW

**Review by phil-davis — APPROVED** [review]
LGTM if the test now passes reliably.

**Comment by saw-jan** [comment]
Still failing: https://drone.owncloud.com/owncloud/web/46187/14/9

File: tests/e2e/cucumber/steps/api.ts [hunk 1]
```diff
@@ -138,12 +138,14 @@ Given(
...
   async function (this: World, stepUser: string, numberOfFiles: number): Promise<void> {
     const user = this.usersEnvironment.getUser({ key: stepUser })
+    const { page } = this.actorsEnvironment.getActor({ key: stepUser })
     for (let i = 1; i <= numberOfFiles; i++) {
       await api.dav.uploadFileInPersonalSpace({
         user,
         pathToFile: `testfile${i}.txt`,
         content: `This is a test file${i}`
       })
+      await page.waitForTimeout(100)
```
Comment (saw-jan) [inline tests/e2e/cucumber/steps/api.ts:148]: NOTE: just to check if waiting between requests helps

**Comment by S-Panta** [comment]
I think this PR can be closed because the issue is solved with https://github.com/owncloud/web/pull/11368

**Pull request closed without merging**